Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -144,6 +155,13 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
             collectionView.contentInset.bottom += typingIndicatorViewHeight
         }
         
+        view.addSubview(scrollToLatestMessageButton)
+        collectionView.bottomAnchor.pin(equalToSystemSpacingBelow: scrollToLatestMessageButton.bottomAnchor).isActive = true
+        scrollToLatestMessageButton.trailingAnchor.pin(equalTo: view.layoutMarginsGuide.trailingAnchor).isActive = true
+        scrollToLatestMessageButton.widthAnchor.pin(equalToConstant: 40).isActive = true
+        scrollToLatestMessageButton.heightAnchor.pin(equalToConstant: 40).isActive = true
```

Comment:
nit: if the goal is to have a circular button, it's better to do:
```swift
x.widthAnchor.pin(equalTo: x.heightAnchor)
x.widthAnchor.pin(equalToConstant: 40)
```
because users will be able to change the size of the button just by overriding a single value